pkg/endpointmanager/manager_test.go

```diff
@@ -950,10 +951,21 @@ func TestMissingNodeLabelsUpdate(t *testing.T) {
 	// Update node labels and verify that the node labels are updated correctly even if the old
 	// labels {k1=v1} are not present in the endpoint manager's state.
 	mgr.localNodeStore.Update(func(ln *node.LocalNode) { ln.Labels = map[string]string{"k2": "v2"} })
-	hostEP, ok := mgr.endpoints[hostEPID]
-	require.True(t, ok)
-	got := hostEP.GetOpLabels()
-	require.Equal(t, []string{"k8s:k2=v2"}, got)
+	require.Eventually(
```

Comment:
Nit. `EventuallyWithT` leads to more expressive failure messages, rather than just expected true, got false. Same for all occurrences.

Reply:
Doh, didn't know this existed! Will start using it always now.